driver.go

```diff
@@ -206,9 +216,45 @@ func newConnector(d *Driver, dsn string) (*connector, error) {
 			config.MaxOpened = val
 		}
 	}
-	if strval, ok := connectorConfig.params["writesessions"]; ok {
-		if val, err := strconv.ParseFloat(strval, 64); err == nil {
-			config.WriteSessions = val
+	if strval, ok := connectorConfig.params["numchannels"]; ok {
+		if val, err := strconv.Atoi(strval); err == nil && val > 0 {
+			config.NumChannels = val
+		}
+	}
+	if strval, ok := connectorConfig.params["rpcpriority"]; ok {
+		var priority spannerpb.RequestOptions_Priority
+		switch strings.ToUpper(strval) {
+		case "LOW":
+			priority = spannerpb.RequestOptions_PRIORITY_LOW
+		case "MEDIUM":
+			priority = spannerpb.RequestOptions_PRIORITY_MEDIUM
+		case "HIGH":
+			priority = spannerpb.RequestOptions_PRIORITY_HIGH
+		default:
+			priority = spannerpb.RequestOptions_PRIORITY_UNSPECIFIED
+		}
+		config.ReadOptions = spanner.ReadOptions{Priority: priority}
```

Comment:
Should we assign a default `ReadOptions` when creating [config](https://github.com/googleapis/go-sql-spanner/blob/0ab64476afb329da8ba981c2b5f456a6122438a7/driver.go#L196) something like `spanner.ReadOptions{}` and then just assign user configured `priority` here, instead of overwriting?
Otherwise when user wants to configure `RequestTag` in ReadOptions it gets overwritten.
Same for others also `TransactionOptions` and `QueryOptions`

Correct me if i am missing something here.

Reply:
I guess we can directly do `config.ReadOptions.Priority = priority` because `config.ReadOptions` will be having default `ReadOptions` property set by default and will not be `nil`.